Content.Server/Fluids/EntitySystems/PuddleSystem.cs

```diff
@@ -512,36 +514,29 @@ public Solution GetOverflowSolution(EntityUid uid, PuddleComponent? puddle = nul
         if (solution.Volume == 0)
             return false;
 
-        var targets = new List<EntityUid>();
-        var reactive = new HashSet<Entity<ReactiveComponent>>();
-        _lookup.GetEntitiesInRange(coordinates, 1.0f, reactive);
-
-        // Get reactive entities nearby--if there are some, it'll spill a bit on them instead.
-        foreach (var ent in reactive)
+        var targets = _lookup.GetEntitiesInRange(coordinates, 1.0f, LookupFlags.Uncontained);
+        foreach (var ent in targets)
```

Comment:
Use the override that takes in a hashset and just store it on a field and re-use it instead please.

Reply:
there is no version that takes EntityCoordinates and hashset